src/pick/__init__.py

```diff
@@ -48,6 +49,22 @@ def __post_init__(self) -> None:
             )
 
         self.index = self.default_index
+        self.parse_options()
+
+    def parse_options(self) -> None:
+        if isinstance(self.options, dict):
+            options: List[Option] = []
+
+            for option, description in self.options.items():
+                if description == "":
+                    options.append(Option(option))
+                else:
+                    options.append(Option(option, description=description))
+
+            self.options = options
+
+        else:
+            pass
```

Comment:
The `else` branch is unnecessary and should be removed. Additionally, I would like to apply the [early return principle](https://dev.to/shameel/-clean-coding-tip-early-return-principle-47kj) to make the code cleaner.